### PR TITLE
add image building

### PR DESCRIPTION
**Pull request by deads2k**
Add image building and prove that travis runs.

### CODE REVIEW

**Comment by spadgett** [comment]
@jwforres

**Comment by deads2k** [comment]
@spadgett I think this is where I leave it for now. 

 1. CI with go build and go test
 2. make target to build a basic image
 3. vendoring origin with glide

Steps after this:
 1. build RPM (you were getting up with @bparees as I recall)
 2. build a real image (again, getting up with ben)
 3. CI integration with origin (probably need to get up with @stevekuznetsov )
 4. jenkins job and bindata stuff (I didn't try to bring it forward)

File: hack/build-image.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+OS_ROOT=$(dirname ${BASH_SOURCE})/..
+
+# Register function to be called on EXIT to remove generated binary.
+function cleanup {
+  rm "${OS_ROOT}/images/origin-web-console-server/bin/origin-web-console-server"
+}
+trap cleanup EXIT
+
+cp -v ${OS_ROOT}/_output/bin/origin-web-console-server "${OS_ROOT}/images/origin-web-console-server/bin/origin-web-console-server"
```
Comment (spadgett) [inline hack/build-image.sh]: Do we need to make sure the directory exists?

```
mkdir -p "${OS_ROOT}/images/origin-web-console-server/bin"
```

Comment (deads2k) [reply hack/build-image.sh]: done

**Review by spadgett — APPROVED** [review]
Thanks @deads2k. I was able to build and run it successfully.

LGTM

**Pull request merged**